src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -158,8 +158,8 @@ function MoneyRequestView({report, betas, parentReport, policyCategories, should
                     titleIcon={Expensicons.Checkmark}
                     description={description}
                     titleStyle={styles.newKansasLarge}
-                    interactive={canEdit}
-                    shouldShowRightIcon={canEdit}
+                    interactive={canEdit && !isSettled}
```

Comment:
so that means that we have two types of the report being settled? Bc canEdit should be false if isReportSettled is true (so canEdit already includes being settled or not)🤔

Reply:
Let me clarify, if a report is settled, an **admin** still can edit it. There is an exception for 3 values of the report: `amount`, `currency` and `date`. So, I added `canEdit && !isSettled` only for `amount` and `date` in the file. For the other values I have `canEdit` only.

Reply:
yeah, I get it, but what I meant is that in canEditMoneyRequest it's already checked if the request is settled or not 
```    const isReportSettled = isSettled(moneyRequestReport.reportID);
    const isAdmin = isExpenseReport(moneyRequestReport) && lodashGet(getPolicy(moneyRequestReport.policyID), 'role', '') === CONST.POLICY.ROLE.ADMIN;
    const isRequestor = currentUserAccountID === reportAction.actorAccountID;
    return !isReportSettled && (isAdmin || isRequestor);

    if (isAdmin) {
        return true;
    }

    return !isReportSettled && isRequestor;
```
and it is also checked here separately
`const isSettled = ReportUtils.isSettled(moneyRequestReport.reportID);`
so canEdit already has the ifo inside if the request is settled or not (if not can edit would be true)
so it's kindof doubling checking the condition
Am I right, or I'm still missing something here?

Reply:
I think it would be clearer to move this logic to `canEdit` since you cannot edit if the report is paid for exampe

Reply:
As you can see, `canEditMoneyRequest` has a condition `isAdmin` to check first. 

So, when a current user is an **admin**, the app ignores `isReportSettled` in general. It returns `true` before checks on settled. Because when the user is admin, he/she can edit. 

However, there is an exception for the 3 values I mentioned above.